File: COT_TOT_PYTHON/input_path/black_input_path/bug2.py
"""
Buggy NON-MODULAR snippet from black, bug ID 2.
"""
def buggy_function(rs, data, freq=None):
        is_fmt_on = False
            for comment in list_comments(container.prefix, is_endmarker=False):
                if comment.value in FMT_ON:
                    is_fmt_on = True
                elif comment.value in FMT_OFF:
                    is_fmt_on = False
            if is_fmt_on:
            yield container
            container = container.next_sibling
    return rs

File: COT_TOT_C/agent4_results_cot/black_agent4_output_code/bug2.c
#include <stdio.h>
#include <string.h>
int buggy_function(int rs, char *data, int freq)
{
    int is_fmt_on = 0;
    char *comments[] = {"FMT_ON", "FMT_OFF", "TEXT"};
    int i;
    for(i=0;i<3;i++)
    {
        if(strcmp(comments[i],"FMT_ON")==0)
            is_fmt_on = 1;
        else if(strcmp(comments[i],"FMT_OFF")==0)
            is_fmt_on = 0;
    }
    if(is_fmt_on)
        printf("Formatting ON\n");
    return rs;
}